elitzur-avro/src/main/scala/com/spotify/elitzur/converters/avro/dynamic/DynamicAccessorCompanion.scala

```diff
@@ -37,20 +37,24 @@ class DynamicAccessorCompanion[T: TypeTag, LT <: BaseValidationType[T]: ClassTag
   private val companion: BaseCompanion[T, LT] =
     implicitly[SimpleCompanionImplicit[T, LT]].companion
   private[dynamic] val validationType: String = companion.validationType
-  private def parseUnsafe(data: Any): Any = companion.parse(data.asInstanceOf[T])
+  private def parseUnsafe(v: Any): Any = companion.parse(v.asInstanceOf[T])
 
-  private def parseAvro(o: Any): Any =
+  // The transient lazy val is required here for lambda serialization. Details on the use of
+  // transient lazy val can be read here: https://www.lyh.me/lambda-serialization.html#.YsV9Z-zMJ6o
```

Comment:
I'm not sure this link is particularly helpful here - it's oriented around fixing `NotSerializableException`s, which doesn't seem particularly relevant

Reply:
I agree, will remove. I was generally avoiding using `@transient lazy` in the codebase and, at the time, felt adding an explanation why this was required was a good practice. But it seems out-of-place now.